core/arch/arm/include/mm/core_mmu_arch.h

```diff
@@ -7,8 +7,10 @@
 #define __CORE_MMU_ARCH_H
 
 #ifndef __ASSEMBLER__
+#include <arm.h>
 #include <assert.h>
 #include <compiler.h>
+#include <config.h>
```

Comment:
Nitpick: this include belongs in commit "core: split core/arch/arm/mm/core_mmu.c" rather than "core: add core_mmu_table_write_barrier()".

Reply:
I'll fix.